apps/swirl-docs/src/pages/api/tokens/v0/index.ts

```diff
@@ -0,0 +1,11 @@
+// Next.js API route support: https://nextjs.org/docs/api-routes/introduction
+import type { NextApiRequest, NextApiResponse } from "next";
+import { OVERVIEW_HTML } from "./utils";
+
+export default function handler(
+  req: NextApiRequest,
+  res: NextApiResponse<any>
+) {
+  res.setHeader("Content-Type", "text/html");
+  res.status(200).send(OVERVIEW_HTML);
+}
```

Comment:
**suggestion:** Would be nice if all endpoints (including the overview) would respond with JSON as default.

Reply:
would agree, but other requirements were [set](https://flipapp.myjetbrains.com/youtrack/issue/APP-11253/Tokens-via-API#focus=Comments-4-21567.0-0) ^-^